### ISSUE TITLE
Internal Server Error 500 on video details page

### ISSUE THREAD

**Issue by ziggca** [bbilly1/tubearchivist#109]
On some videos I get the following error on video details page. Maybe they don't have a rating?

```
Internal Server Error: /video/MyhYYxGT5d4/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 98, in dispatch
    video_data["stats"]["average_rating"] = self.star_creator(rating)
  File "/app/./home/views.py", line 892, in star_creator
TypeError: '>=' not supported between instances of 'NoneType' and 'float'
[pid: 27|app: 0|req: 32/32] 172.18.0.1 () {46 vars in 850 bytes} [Wed Dec 15 22:25:37 2021] GET /video/MyhYYxGT5d4/ => generated 85577 bytes in 195 msecs (HTTP/1.1 500) 6 headers in 186 bytes (1 switches on core 0)

```

**Comment by bbilly1** [comment]
Thanks for pointing it out, and yes, you are right, looks like the dislikes and by extension the average rating are now completely gone, hence the error you see in the `star_creator`. Fix is already incoming: [53f93a7](https://github.com/bbilly1/tubearchivist/commit/53f93a7bb9da88c32153371c7d4426fe261fd530).

**Comment by ziggca** [comment]
Ah the infamous thumbs down removal... looking forward to a new release, great work!

**Comment by bbilly1** [comment]
This is now merged in v0.0.9. Please update.
